src/co_refract.jl

```diff
@@ -0,0 +1,159 @@
+# This file is a part of AstroLib.jl. License is MIT "Expat".
+
+function _co_refract(old_alt::T, altitude::T, pressure::T, temperature::T,
+                    epsilon::T, to_observe::Bool) where {T<:AbstractFloat}
+
+    if isnan(temperature)
+        if altitude > 11000
+            temperature = 211.5
```

Comment:
If `T != Float64`, `temperature` will be type-unstable here.  Convert to `T` type with `T(211.5)` ;-)